tictactoe/test/tictactoe_test.exs

```diff
@@ -4,11 +4,23 @@ defmodule TictactoeTest do
   alias Tictactoe.Game
 
   test "new game returns structure" do
-    game = Game.new_game()
+    game = Game.new_game
 
-    assert game.current_player == "X"
     assert game.game_state == "playing"
+    assert game.current_player == %Player.Human{mark: "X"}
     assert game.board == [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    assert game.player_x == %Player.Human{mark: "X"}
+    assert game.player_o == %Player.Human{mark: "O"}
```

Comment:
It feels like you have a lot of String data structure through your application - If I want to change the X by a T, it would mean that you have to update it through the all application, what do you think about abstracting that to one place? ex Mark.cross/Mark.round